### PR TITLE
Adds banner for KubeCon + CloudNativeCon EU 2022

### LINKED ISSUE #33560: Add banner for KubeCon 2022 EU
**This is a Feature Request**

**What would you like to be added**
Set a banner to announce KubeCon 2022 EU

**Why is this needed**
We usually turn on a banner for the upcoming KubeCon.
Logos are available from: https://github.com/cncf/artwork/tree/master/other/kubecon-cloudnativecon/2022-eu (make sure to copy the logo into this repo)
https://events.linuxfoundation.org/kubecon-cloudnativecon-europe/ should help with text, or CNCF staff may be able to advise. Also see the [banner](https://github.com/kubernetes/website/blob/85cbd9d9e5ca49b44b91a3ffb29b99adf0ac4761/data/announcements/scheduled.yaml#L74-L83) config for last year's EU KubeCon.

The HTML colors seem to be #000000, #ec6b43 and #fde1cb.

**Comments**
This change will need steering committee approval.
I would set the banner to start on the 9th of May and to end on the 20th of May at 1800 CEST.

/priority important-soon

### PR DESCRIPTION
**Pull request by Babapool**
Fixes #33560 

This PR:
 - Adds banner for KubeCon + CloudNativeCon EU 2022 on the K8s website.
 - Changes the end time of the _**Dockershim Removal**_  to match the start time of the KubeCon Banner.
 
image

I am open to suggestions on the colour scheme (I do not have a good sense of colour schemes😅😅)

Files changed:
- https://github.com/kubernetes/website/blob/main/data/announcements/scheduled.yaml
- https://github.com/kubernetes/website/tree/main/static/images/announcements (to add the banner image)

Signed by: Vitthal Kaul vitthalsai2001@gmail.com

### CODE REVIEW

File: data/announcements/scheduled.yaml [hunk 1]
```diff
@@ -52,7 +52,7 @@ announcements:
 
 - name: Kubecon 2020 EU
   startTime: 2020-08-16T00:00:00 # Added in https://github.com/kubernetes/website/pull/23183
-  endTime: 2020-08-20T00:00:00 # Removed in https://github.com/kubernetes/website/pull/23272
+  endTime: 2020-08-20T00:00:00 # Removed in https://github.com/kubernetes/website/pull/23272ll
```
[the remark this answers is not in the document]
Comment (Babapool) [reply data/announcements/scheduled.yaml]: While copying, by mistake the URL must have changes 😅😅. I will do the needful. Thanks :)

File: data/announcements/scheduled.yaml [hunk 2]
```diff
@@ -103,3 +103,14 @@ announcements:
   message: |
     2 days of incredible opportunities to collaborate, learn + share with the entire community!<br />
     December 9 + 10, 2021.
+
+- name: Kubecon 2022 EU
+  startTime: 2022-05-09T00:00:00  #Added in https://github.com/kubernetes/website/pull/33562
+  endTime: 2022-05-20T18:00:00
+  style: "background: linear-gradient(90deg, rgb(253, 225, 203) 0%, rgb(236, 107, 67) 100%)"
```
Comment (sftim) [inline data/announcements/scheduled.yaml:110]: I would use black text, not white. The white text is too low contrast against the background.

Also, move the blend point. Here's how that might look:
```suggestion
  style: >-
    background: linear-gradient(90deg, rgb(253, 225, 203) 0%, rgb(236, 107, 67) 75%, rgb(236, 107, 67) 100%);
    color: #000;
```

File: data/announcements/scheduled.yaml [hunk 3]
```diff
@@ -32,7 +32,7 @@ announcements:
 
 - name: Dockershim removal
   startTime: 2022-04-07T00:00:00 # Added in PR: https://github.com/kubernetes/website/pull/32805
-  endTime: 2022-05-10T00:00:00 # Expires 2022-05-10, but will change text slightly on release date
+  endTime: 2022-05-09T00:00:00 # Expires 2022-05-10, but will change text slightly on release date
```
Comment (sftim) [inline data/announcements/scheduled.yaml:35]:
```suggestion
  endTime: 2022-05-09T00:00:00
```

**Comment by Babapool** [comment]
@sftim I tried the workflow you suggested, but it is asking me to open a new PR with 3 commits. What should I do, could you please guide me. Thanks.

**Comment by Babapool** [comment]
As I had some issues in rebasing this PR based on @sftim's suggestion I have opened a new PR #33578 . Thanks

**Comment by Babapool** [comment]
I will close this PR now.
/close

**Pull request closed without merging**